.eslintrc.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
```

Comment:
Make sure you're checking with the latest version: I just added them as dependencies.